.gitignore

```diff
@@ -172,4 +172,7 @@ msbuild.binlog
 BenchmarkDotNet.Artifacts
 
 artifacts
-.assets
\ No newline at end of file
+.assets
+
+# Temporary cadl folders for cadl generation
+TempCadlFiles/
```

Comment:
Any reason to put these in a new top level folder instead of under artifacts like everything else?

Reply:
not sure I follow, they are currently inside each project folder, I have the ignore so they are never accidentally checked in.

Reply:
It's because `cleanup: false` in `cadl-location.yaml`. I remove that property in another PR, and `TempCadlFiles` will be removed after `dotnet build /t:GenerateFiles`